notebooks/2_4_highlight_fragments_in_ligand.ipynb

```diff
@@ -8,8 +8,6 @@
     "\n",
```

Comment:
<p>You can just delete the commented code</p><p> </p><p>And maybe you can add a small note that if the code in this notebook is not working, they should check the rdkit version (in case they still have the an older version/environment) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/KinFragLib/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>